catalog.html

```diff
@@ -0,0 +1,128 @@
+          <!-- Здесь будет блок сортировки -->
+          <ul class="template-examples-list">
+             <li>
+               <div class="template-example">
+                 <img src="http://placehold.it/150x100" alt="Sedona">
+                 <p class="template-example-content">
+                   <b>Sedona</b>
+                   <span>Информационный сайт для туристов</span>
+                   <span class="button template-example-button">9 900 руб.</span>
+                 </p>
+               </div>
+             </li>
+             <li>
+               <div class="template-example">
+                 <img src="http://placehold.it/150x100" alt="Pink App">
+                 <p class="template-example-content">
+                   <b>Pink App</b>
+                   <span>Продуктовый лендинг приложения для iOS и Android</span>
+                   <span class="button template-example-button">9 900 руб.</span>
+                 </p>
+               </div>
+             </li>
+             <li>
+               <div class="template-example">
+                 <img src="http://placehold.it/150x100" alt="Barbershop">
+                 <p class="template-example-content">
+                   <b>Barbershop</b>
+                   <span>Сайт салона красоты. Для мужчин, да.</span>
+                   <span class="button template-example-button">9 900 руб.</span>
+                 </p>
+               </div>
+             </li>
+             <li>
+               <div class="template-example">
+                 <img src="http://placehold.it/150x100" alt="Mishka">
+                 <p class="template-example-content">
+                   <b>Mishka</b>
+                   <span>Интернет-магазин вязаных игрушек</span>
+                   <span class="button template-example-button">9 900 руб.</span>
+                 </p>
+               </div>
+             </li>
+             <li>
+               <div class="template-example">
+                 <img src="http://placehold.it/150x100" alt="A Plus">
+                 <p class="template-example-content">
+                   <b>A Plus</b>
+                   <span>Лендинг курсов повышения квалификации</span>
+                   <span class="button template-example-button">9 900 руб.</span>
+                 </p>
+               </div>
+             </li>
+             <li>
+               <div class="template-example">
+                 <img src="http://placehold.it/150x100" alt="Кваст">
+                 <p class="template-example-content">
+                   <b>Кваст</b>
+                   <span>Промо-сайт производителя крафтового кваса</span>
+                   <span class="button template-example-button">9 900 руб.</span>
+                 </p>
+               </div>
+             </li>
+           </ul>
+        </div>
+      </section>
+      <!-- Здесь будет блок пагинации и контакты -->
+    </main>
+    <footer class="main-footer">
+      <div class="footer-social">
```

Comment:
Не уверен в необходимости данного дива. 
![image](https://user-images.githubusercontent.com/11176813/51804270-7bd18e00-2280-11e9-8ae0-3d823f9c691a.png)

```suggestion
   <ul  class="footer-social">
          <li><a href="https://vk.com">VK</a></li>
          <li><a href="https://www.facebook.com">facebook</a></li>
          <li><a href="https://www.instagram.com">instagram</a></li>
   </ul>
```

Можно оставить, проверишь при написании CSS